TASVideos.WikiEngine/Util.cs

```diff
@@ -72,6 +72,25 @@ public static async Task RenderTextAsync(string content, TextWriter w, IWriterHe
 		}
 	}
 
+	public static async Task RenderMetaDescriptionAsync(string content, StringBuilder sb, IWriterHelper h)
+	{
+		List<INode> results;
+		try
+		{
+			results = NewParser.Parse(content);
+		}
+		catch (NewParser.SyntaxException e)
+		{
+			results = Builtins.MakeErrorPage(content, e);
+		}
+
+		var ctx = new WriterContext(h);
+		foreach (var r in results)
```

Comment:
We should break this loop early once the `sb` is full.  Most pages will have a lot of their content directly under the top level list, so stopping early here could save time.